### DISCUSSION TITLE
locally get secret code

### DISCUSSION THREAD

**Discussion by dnurgeldiyev** [zitadel/zitadel#2566] [Q&A] [answered]
I installed it locally, everything went well, but after registration how can I get the secret code to which it should be sent by mail? the code is not there or is there another way?

**Comment by hifabienne** [comment, marked as answer]
Hi @dnurgeldiyev 
To receive the code you need to configure a valid smtp or a chat channel. (We use google chat) In case of a chat provider you need to enable the debugmode. You can find this configurations in the system-defaults.yaml
